pkg/descriptions.json

```diff
@@ -455,6 +455,19 @@
     ]
   },
   {
+    "Name": "System.Data.Odbc",
+    "Description": "Provides a collection of classes used to access an ODBC data source in the managed space",
```

Comment:
I'm not sure what `in the managed space` means but I'd probably write `from managed code` ...

Reply:
This is more or less what the MSDN page says about the namespace: https://msdn.microsoft.com/en-us/library/system.data.odbc(v=vs.110).aspx

Reply:
ha, fine